src/popup/router/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -1,24 +1,53 @@
 <template>
   <Modal
-    class="confirm-tx-sign-modal"
+    expanded
+    fixed-footer
+    class="confirm-transaction-sign"
     data-cy="popup-aex2"
   >
-    <SignAccountIdenticons
-      v-bind="transaction"
-      slot="header"
-    />
+    <TransactionOverview :tx="transaction" />
 
-    <SignDetailsList :transaction="transaction" />
+    <div class="details">
+      <DetailsItem :label="$t('pages.signTransaction.fee')">
+        <TokenAmount slot="value" :amount="+aettosToAe(transaction.fee)" hideFiat />
+      </DetailsItem>
+      <DetailsItem :label="$t('pages.signTransaction.total')">
+        <TokenAmount
+          slot="value"
+          :amount="getTxAmountTotal(transaction)"
+          :symbol="getTxSymbol(transaction)"
+        />
+      </DetailsItem>
+    </div>
+
+    <div class="show-advanced">
+      <button
+        class="toggle"
+        :class="{ active: showAdvanced }"
+        @click="showAdvanced = !showAdvanced"
+      >
+        <Minus v-if="showAdvanced" />
+        <Plus v-else />
+        <span class="text">{{ $t('pages.signTransaction.advanced') }}</span>
+      </button>
+      <CopyButton :value="code" :message="$t('copied')" />
+    </div>
+
+    <div class="code" v-if="showAdvanced">
+      <DetailsItem :value="code" :label="$t('pages.signTransaction.code')" />
```

Comment:
I think we need to redesign and use `SignDetailsList` here. Different transaction types have fields that would be useful to show here, not only the `code`.

Reply:
Which other fields do you think are useful for different transaction types?

Reply:
in Base aepp we were rendering these fields:
https://github.com/aeternity/aepp-base/blob/e25875692ea3f0be766e0911507daed59b8fe05a/src/components/mobile/ConfirmTransactionSignModal.vue#L101-L111
We can add nonce as far as it is displayed in transaction details. In the future, we may add fields of oracle-related transactions, state channels and so.